embulk-input-postgresql/src/main/java/org/embulk/input/PostgreSQLInputPlugin.java

```diff
@@ -49,11 +51,36 @@
         @ConfigDefault("false")
         public boolean getSsl();
 
+        @Config("ssl_version")
+        @ConfigDefault("null")
+        public Optional<SslVersion> getSslVersion();
```

Comment:
In the current implementation, users need to specify "TLSv1_1" or "TLSv1_2". Cannot use "TLSv1.1" or "TLSv1.2". I think that the usage of dot separator (".") looks reasonable for users.